packages/ckeditor5-dev-utils/lib/translations/multiplelanguagetranslationservice.js

```diff
@@ -239,6 +235,30 @@ module.exports = class MultipleLanguageTranslationService extends EventEmitter {
 		];
 	}
 
+	/**
+	 * @param {Object} options
+	 * @param {String} options.outputDirectory Output directory for the translation files relative to the output.
+	 * @param {String[]} options.compilationAssetNames Original asset names from the compiler (e.g. Webpack).
```

Comment:
`s/String[]/Array.<String>`

Reply:
Missing the `@returns` tag.

Reply:
Not sure if we document the `return` everywhere. Especially for such places that the code is rather internal. I can specify only the type as the function name suggests what it returns.

Reply:
If it returns Object, just put `@returns {Object}`. We don't need digging into its details.